mmdeploy/apis/pytorch2onnx.py

```diff
@@ -84,9 +84,10 @@ def torch2onnx(img: Any,
     from mmdeploy.apis import build_task_processor
     task_processor = build_task_processor(model_cfg, deploy_cfg, device)
 
-    torch_model = task_processor.init_pytorch_model(model_checkpoint)
+    torch_model = task_processor.init_pytorch_model(
+        model_checkpoint, is_onnx_export=True)
     data, model_inputs = task_processor.create_input(img, input_shape)
-    if not isinstance(model_inputs, torch.Tensor):
+    if not isinstance(model_inputs, torch.Tensor) and len(model_inputs) == 1:
```

Comment:
`torch2onnx_impl` requires `Tensor` type for `input` argument. If you want to feed tuple(Tensor,...), you may have to update `torch2onnx_impl`